backend/test/Test/UploadSpec.hs

```diff
@@ -121,6 +126,25 @@ spec = withContext $ startWithInitial $ do
         Library.getCompounds noSorting fullContent >>= shouldBeZeroLength
         Dashboard.getExperiments Nothing Nothing Nothing noSorting fullContent
           >>= shouldBeZeroLength . erExperiments
+
+    -- Note that there are some checks in LibrarySpec and DashboardSpec as well.
```

Comment:
Erm, did you copy this comment intentionally? I mean, there are some tests with file upload where compounds are added, but we don't check anything about mde link generation there :shrug: